components/Ink/src/MDCInkView.m

```diff
@@ -112,69 +110,38 @@ - (void)traitCollectionDidChange:(UITraitCollection *)previousTraitCollection {
 
 - (void)setInkStyle:(MDCInkStyle)inkStyle {
   _inkStyle = inkStyle;
-  if (self.usesLegacyInkRipple) {
-    switch (inkStyle) {
-      case MDCInkStyleBounded:
-        self.inkLayer.masksToBounds = YES;
-        self.inkLayer.bounded = YES;
-        break;
-      case MDCInkStyleUnbounded:
-        self.inkLayer.masksToBounds = NO;
-        self.inkLayer.bounded = NO;
-        break;
-    }
-  } else {
-    switch (inkStyle) {
```

Comment:
Thinking aloud here:
So removing this causes the maxRippleRadius to be ignored.

We dont think clients will be relying on this?

Reply:
I don't think we're breaking anyone. I was able to make the setter a _lot_ less clever by making the getter a _bit_ more clever. With this PR calling setMaxRippleRadius will just set it on the legacy ink layer, no matter what. It won't set it to 0 if it's using the new ink and its bounded. However, in the "startTouchAtPoint blah blah blah" method where the ink is actually implemented the value for maxRippleRadius that's read ends up being 0.